pkg/integrations/manager.go

```diff
@@ -68,6 +70,10 @@ type ManagerConfig struct {
 	// ListenHost tells the integration Manager which port the Agent is
 	// listening on for generating Prometheus instance configs
 	ListenHost *string `yaml:"-"`
+
+	TLSConfig config_util.TLSConfig `yaml:"http_tls_config"`
+
+	SeverUsingTLS bool `yaml:"-"`
```

Comment:
Can you add a comment saying why this would be set to true?

Reply:
(I feel a little weird about how many things we're copying from the server config to the integrations manager config, but I can't see a better way of doing it than what you have here)